source/classes.tex

```diff
@@ -6030,7 +6030,7 @@
 struct X { X(A*); };
 
 struct E : C, D, X {
-  E() : D(this),    // undefined behavior: upcast from \tcode{E*} to \tcode{A*} might use path \tcode{E*} $\rightarrow$ \tcode{D*} $\rightarrow$ \tcode{A*}
+  E() : D(this),    // undefined behavior: upcast from \tcode{E*} to \tcode{A*} can use path \tcode{E*} $\rightarrow$ \tcode{D*} $\rightarrow$ \tcode{A*}
```

Comment:
ISO doesn't like "could" in notes and examples, I think.

Reply:
It's actually a bit more subtle than that: The verbal forms for *provisions* are strictly controlled. But if we're not expressing a provision, then there's in principle no restriction. What ISO/CS is worried about then is that one might _mistakenly_ misunderstand something to be a provision (even though it's not meant to be one), and they then recommend rephrasing it so that such a misunderstanding can't happen.

We can belabor this separately for the WD, but for the IS I'd be OK to accept a somewhat crude replacement.